invenio_rdm_records/services/config.py

```diff
@@ -455,6 +528,10 @@ class RDMRecordServiceConfig(RecordServiceConfig, ConfiguratorMixin):
             if_=RecordLink("{+api}/records/{id}/media-files"),
             else_=RecordLink("{+api}/records/{id}/draft/media-files"),
         ),
+        "thumbnails": RDMThumbnailLink(
+            "{+api}/iiif/record:{id}:",
+            sizes=record_thumbnail_sizes,
+        ),
```

Comment:
minor: in order to skip the link entirely if there's no thumbnail-able file, we'll need a `when=...` to check for the file. This will also simplify the code for the `expand` method, since you can assume that for this record there will be a file